.github/workflows/wheels.yml

```diff
@@ -12,8 +12,16 @@ on:
     paths:
       - ".github/workflows/wheels.yml"
       - "homeassistant/package_constraints.txt"
-      - "requirements_all.txt"
-      - "requirements.txt"
+      - "requirements_all_wheels_aarch64.txt"
```

Comment:
While it is common for `requirements` files to be in the root folder, this adds 9 files to the 5 already on the root folder (if I counted correctly it will end up in 14 files) wouldn't it be cleaner to move this to a folder `requirements` or `architecture_requirements`?

Reply:
Yes, it's not great and I agree it's too much clutter. An alternative I've discussed with @edenhaus is if the additional files can be generated on the fly by the CI job instead of stored in the repo.